app/views/match_mailer/match_canceled.text.erb

```diff
@@ -1,6 +1,18 @@
 <%= t('mailer.hello', firstname: @user.first_name) %>
 
-<%= t('match_mailer.next_game', name: @match.event.name, team_home: @match.team_home.name, team_away: @match.team_away.name) %>
-<%= t('.info') %>
+<%= t('.info_txt',
+      team_name: @opponent_team.name,
+      event_name: @match.event.name) %>
 
-<%= event_overview_url(@match.event) %>
\ No newline at end of file
+<% if @match.start_time %>
+<%= t('match_mailer.start_time',
```

Comment:
should we indent markup within an `if` block?

Reply:
Please see comment below.